spec/factory_bot/disallows_duplicates_registry_spec.rb

```diff
@@ -1,17 +1,19 @@
 describe FactoryBot::Decorator::DisallowsDuplicatesRegistry do
-  let(:registry) { double("registry", name: "Great thing", register: true) }
-
-  subject { described_class.new(registry) }
-
   it "delegates #register to the registry when not registered" do
+    registry = double("registry", name: "Great thing", register: true)
+    decorator = described_class.new(registry)
```

Comment:
I remember @purinkle having negative opinions about `described_class` although I can't remember the exact reasoning why.

Reply:
Looks like there's some discussion around deprecating it, due to some people being confused by its API:

https://blog.rubystyle.guide/rspec/2019/08/04/rspec-described_class.html

If true, that could be a big reason not to use it.

Reply:
I've been trying to replace it with inlined classes in other refactorings I've done (forgot to do so here but will change it).  I'll probably double down on doing so after reading the above links.